keyboards/pursuit40/keymaps/via/keymap.c

```diff
@@ -0,0 +1,100 @@
+        KC_TILD, KC_EXLM, KC_AT,   KC_HASH, KC_DLR,  KC_PERC, KC_CIRC, KC_AMPR, KC_ASTR, KC_LPRN, KC_RPRN, KC_BSPC,
+        KC_SLSH, KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_BSLS,
+        _______, _______, _______, _______, _______, _______, _______, KC_PPLS, KC_EQL,  _______, _______, _______,
+        KC_ESC,  CALTDEL, TSKMGR,           _______,                   _______,          KC_NUBS, KC_GRV,  _______
+    ),
+
+    /* [2]
+     * ,--------------------------------------------------------------------------------------.
+     * |------------+------+------+------+------+------+------+------+------+------+------+---|
+     * |       |      |      |  UP  |       |      |     |   _   |     |   [   |  ]    |      |
+     * |------------+------+------+------+------+-------------+------+------+------+------+---|
+     * |       |     | LEFT | DOWN  | RIGHT |     |      |   -  |      |   [   |   ]   |      |
+     * |------------+------+------+------+------+-----+-----+------+------+------+------+-----|
+     * |   F1  | F2 |  F3  |   F4  | F5    | F6  | F7   | F8   | F9    | F10  | F11   | F12   |
+     * |-------+-------+-------+-------+-------+-------+------+------+------+------+------+---|
+     * |     RESET    |           |          |         |         |         |        |         |
+     * `---------------------------------------------------------------------------------------'
+     */
+    [2] = LAYOUT_default(
+        _______, _______, _______, KC_UP,   _______, _______, _______, KC_UNDS, _______, KC_LBRC, KC_RBRC, _______,
+        _______, _______, KC_LEFT, KC_DOWN, KC_RGHT, _______, _______, KC_MINS, _______, KC_LCBR, KC_RCBR, _______,
+        KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,
+        RESET,   _______, _______,          _______,                   _______,          _______, _______, _______
+    ),
+
+ /* [3]
+     * ,---------------------------------------------------------------------------------------.
+     * |------------+------+------+------+------+------+------+------+------+------+------+----|
+     * |      |     |      |      |      |      |      |     |     |      |      |      |      |
+     * |------------+------+------+------+------+-------------+------+------+------+------+----|
+     * |      |     |      |      |      |      |      |     |     |      |      |      |      |
+     * |------------+------+------+------+------+------|------+------+------+------+------+----|
+     * |      |     |      |      |      |      |      |     |     |      |      |      |      |
+     * |-------+-------+-------+-------+-------+-------+------+------+------+------+------+----|
+     * |      |     |      |      |      |      |      |     |     |      |      |      |      |
+     * `---------------------------------------------------------------------------------------'
+     */
+    [3] = LAYOUT_default(
```

Comment:
Looks like this layer has too many items.

Reply:
should be fixed now